scripts/fbp-runner.sh

```diff
@@ -19,21 +19,22 @@ SCRIPT="$3"
 ENV_PATH="$2"
 SERVICE="fbp-runner@"$(systemd-escape $ENV_PATH)
 
-systemctl stop $SERVICE
+if [ $1 == "stop" ]; then
+	systemctl stop $SERVICE
 
-if [ $1 == "start" ]; then
+elif [ $1 == "start" ]; then
```

Comment:
if it was already running, the old behavior would stop the old then run the new. The new behavior will keep the old one running.

Maybe go with `restart`?